BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -354,19 +354,38 @@ extension AttributedLabel {
                 return []
             }
 
-            func alignmentMultiplier() -> CGFloat {
-                // If a paragraph style spans the range of the text, use its alignment.
-                // Otherwise, fall back to `.natural`.
-                let alignment: NSTextAlignment = {
-                    var range: NSRange = NSRange(location: -1, length: 0)
-                    let style = textStorage.attribute(.paragraphStyle, at: 0, effectiveRange: &range)
-
-                    guard let style = style as? NSParagraphStyle, range == textStorage.entireRange else {
-                        return .natural
+            /// The below positioning calculation assumes that there is only one
+            /// alignment within the label, so verify that is the case.
+            func alignment() -> NSTextAlignment {
+
+                guard let string = attributedText, string.length > 0 else {
+                    return textAlignment
+                }
+
+                var alignments = [NSTextAlignment]()
```

Comment:
Could we use a `Set` here? We really care about _unique_ alignments, right? For example, if there are 3 `paragraphStyle` runs that are all left-aligned, that would be fine.

Reply:
Ahh yeah, good call. Updating!

- [ ] Set

Reply:
Done